pkg/gofr/gofr.go

```diff
@@ -236,8 +236,8 @@ func (a *App) Logger() logging.Logger {
 
 // SubCommand adds a sub-command to the CLI application.
 // Can be used to create commands like "kubectl get" or "kubectl get ingress".
-func (a *App) SubCommand(pattern string, handler Handler) {
-	a.cmd.addRoute(pattern, handler)
+func (a *App) SubCommand(pattern string, handler Handler, description string) {
```

Comment:
This is a breaking change as the SubCommand function takes description as a required argument.
I would recommend to take description as a variadic string(slice of string) and join them with "/n" at end of each string line.

Reply:
Your comment somehow matches mine

https://github.com/gofr-dev/gofr/pull/661#discussion_r1616710862